src/Core/src/Platform/iOS/SliderExtensions.cs

```diff
@@ -44,14 +46,14 @@ public static void UpdateThumbColor(this UISlider uiSlider, ISlider slider)
 		public static async Task UpdateThumbImageSourceAsync(this UISlider uiSlider, ISlider slider, IImageSourceServiceProvider provider)
 		{
 			var thumbImageSource = slider.ThumbImageSource;
-
 			if (thumbImageSource != null)
 			{
 				var service = provider.GetRequiredImageSourceService(thumbImageSource);
 				var scale = uiSlider.GetDisplayDensity();
 				var result = await service.GetImageAsync(thumbImageSource, scale);
-				var thumbImage = result?.Value;
-
+				var thumbImageSize = result?.Value.Size ?? CGSize.Empty;
```

Comment:
Does it make sense if the ThumbImageSize is Empty to still try to resize the image, or in that case we should go to the case where we use `var thumbImage = result?.Value;`